app/controllers/users_controller.rb

```diff
@@ -12,4 +13,22 @@ def show
   def new
     @message = Message.new
   end
+
+  # POST /users or /users.json
+  # def create
+  #   @user = User.new(user_params)
+
+  #   respond_to do |format|
+  #     if @user.save
+  #       # Tell the UserMailer to send a welcome email after save
+  #       UserMailer.with(user: @user).welcome_email.deliver_later
+
+  #       format.html { redirect_to(@user, notice: 'User was successfully created.') }
+  #       format.json { render json: @user, status: :created, location: @user }
+  #     else
+  #       format.html { render action: 'new' }
+  #       format.json { render json: @user.errors, status: :unprocessable_entity }
+  #     end
+  #   end
+  # end
```

Comment:
Hi @elisha2kyakpo1,
When a user inputs their details on their sign up page, nothing really happens, they don't get redirected anywhere until they've confirmed their email address. However, their inputs are viewable in rails server. Could you please explain a little bit more on what you expect to happen in case we're missing something? Also, you can see the project requirements here: https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md

Reply:
Done - Removed comments

Reply:
hello @elisha2kyakpo1, 

As my TSE colleugue said the data is saved and not being redirected to any place, and also the confirmation like you said it should be in the rails server but there is none  we should see the token link in the rails server to confirm the address and then login